### PR TITLE
Less logging of DstvExceptions to the console

### PR DESCRIPTION
**Pull request by EjaYF**
This fix addresses issues where DstvExceptions appear in the console log unnecessary.

### CODE REVIEW

File: DSTV.Net/Implementations/BodyReader.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public static async Task<IEnumerable<DstvElement>> GetElementsAsync(ReaderContex
             foreach (var holeNote in holeBlocks.SelectMany(holeList => holeList))
                 try
                 {
+                    if (holeNote.Equals(ContourType.BO.ToString(), StringComparison.Ordinal)) continue;
```
Comment (Mstrdk) [inline DSTV.Net/Implementations/BodyReader.cs:20]: I'd prefer the use of the nameof operator in these cases (e.g. `nameof(ContourType.BO)`), but that's pretty personal...

**Review by Mstrdk — APPROVED** [review]

File: DSTV.Net/Implementations/BodyReader.cs (hunk 1 above)
Comment (tluijken) [reply DSTV.Net/Implementations/BodyReader.cs:20]: I agree

File: DSTV.Net/Implementations/BodyReader.cs [hunk 2]
```diff
@@ -55,6 +57,7 @@ public static async Task<IEnumerable<DstvElement>> GetElementsAsync(ReaderContex
             foreach (var bendNote in bendBlocks.SelectMany(bendList => bendList))
                 try
                 {
+                    if (bendNote.Equals(ContourType.KA.ToString(), StringComparison.Ordinal)) continue;
```
Comment (tluijken) [inline DSTV.Net/Implementations/BodyReader.cs:60]: I feel like there is a lot of repetition going on here...should we make a generic function for this?

**Review by tluijken — CHANGES_REQUESTED** [review]

File: DSTV.Net/Implementations/BodyReader.cs (hunk 2 above)
Comment (EjaYF) [reply DSTV.Net/Implementations/BodyReader.cs]: I've simplified the code

File: DSTV.Net/Implementations/BodyReader.cs (hunk 1 above)
Comment (EjaYF) [reply DSTV.Net/Implementations/BodyReader.cs]: Me too!

**Review by tluijken — APPROVED** [review]

**Pull request merged**